Assets/Scripts/UIAndMenus/EscMenuController.cs

```diff
@@ -69,8 +69,8 @@ public void CloseSettings()
         pauseMenuPanel.SetActive(true);
     }
 
-    public void QuitToMainMenu()
-    {
-        savePromptPanel.SetActive(true);
-    }
+    //public void QuitToMainMenu()
+    //{
+    //    savePromptPanel.SetActive(true);
+    //}
```

Comment:
did you comment this out because it saves automatically or was there another reason?

Reply:
I commented it out because it was giving me errors because I didn't have the save prompt serialized, but I couldn't find it anywhere. I meant to ask Joseph about it and forgot. It should absolutely 100% not be kept commented out

Reply:
@jidaprano

Reply:
This should be fixed now